index.rst

```diff
@@ -12,18 +12,633 @@
+    +===========+=============+================+===============+===============+=====================+
+    |   Default |       13244 |           6166 |        4627.0 |       50172.0 |              2391.0 |
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+    |  Pre-Conv |       10909 |           6381 |        4594.0 |       55649.0 |              1732.0 |
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+    | Auto Mode |       13727 |           6698 |        4321.0 |       45728.0 |              2348.0 |
+    +-----------+-------------+----------------+---------------+---------------+---------------------+
+
+We can see that the amount of matches decreases to almost a 50% when applying the flag cuts. Our conclusion is that maybe our set of flags is too agressive, and it is not really discriminating the real transients from the potential artifacts, but instead vettoing on information that might or might not be relevant.
+
+
+CCD Illumination: X-Y distribution of detections
+================================================
+
+Looking at the distribution of the pixel coordinates of the detections we can try to obtain information about the algorithm performance around central and edge areas of the CCD. This contains valuable information about the capability of the algorithm to work in the full field of view and also can be of use when understanding the uniformity of the detections in the pixel field.
+
+.. figure:: /_static/figures/chipillum_x_y_histogram_diffim_flavors.png
+    :name: chipillum_x_y_histogram_diffim_flavors
+    :target: ../_images/chipillum_x_y_histogram_diffim_flavors.png
+    :alt: X-Y pixel coordinate distributions for diaSources
+
+    Distribution of the X and Y coordinates of the diaSources.
+
+Two important caveats of this plot are: the dimensions of the chip are 4096 by 2048, so X axis is smaller, and finally some detection centroids can end up in negative coordinates or, more generally coordinates that exceed the real CCD domain.
+
+As a general note we understand that the number of transiend candidate detections should not depend on the location on the image CCD domain up to a certain extent. This means, there will be some loss of sensitivy on algorithms such as pre-convolution, as the edges will have incomplete information due to kernel padding. In the following figure we can see a zoom in into the interesting edge areas, for x and y axes.
+
+.. figure:: /_static/figures/chipillum_x_y_histogram_zoom_diffim_flavors.png
+    :name: chipillum_x_y_histogram_zoom_diffim_flavors
+    :target: ../_images/chipillum_x_y_histogram_zoom_diffim_flavors.png
+    :alt: X-Y pixel coordinate distributions for diaSources
+
+    Distribution of the X and Y coordinates of the diaSources, zoomed in the edges and central x-axis locations.
```

Comment:
The extra sources at negative coordinates for pre-convolution are alarming. We should look into these on a new ticket. It helps that these are caught with our flag cuts (below), but it's potentially a problem that they are getting those coordinates in the first place.